src/images/Groceries.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="17" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This icon also needs to have the dimensions adjusted. You'll need to resize it down a bit in Figma.